packages/api/src/utils/autocompleteBlockHash.ts

```diff
@@ -0,0 +1,24 @@
+import { prisma } from "@blobscan/db";
+
+/* Autocomplete a block hash from a truncated version of it.
+   @param partialHash - The first bytes of a block hash.
+   @returns The block hash, if there is a single ocurrence, or null.
+ */
+export async function autocompleteBlockHash(partialHash: string) {
+  const block = await prisma.block.findFirst({
```

Comment:
@luis-herasme Can you modify it to log an error if there exists more than one occurrence?

Reply:
Done [a83e1bc](https://github.com/Blobscan/blobscan/pull/608/commits/a83e1bca1e2369a5fe987de0088876c869dd2f25)